packages/ui/src/components/ProTable/Pagination/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement } from 'react';
+import { DoubleLeftOutlined, LeftOutlined, RightOutlined } from '@ant-design/icons';
+import { Button, Select, Space } from 'antd';
+
+import { IProTablePagination } from '../types';
+import { reverseSortDirection } from '../utils';
+
+import { PaginationViewPerQuery } from './constants';
+import { getPaginationOptions } from './utils';
+
+import styles from '@ferlab/style/components/protable/Pagination.module.scss';
+
+const iconSize = 11;
+
+const Pagination = ({
+    current,
+    defaultViewPerQuery,
+    dictionary,
+    onChange,
+    onPageChange,
+    onShowSizeChange,
+    queryConfig,
+    searchAfter,
+    setQueryConfig,
+    total,
+}: IProTablePagination): ReactElement => {
+    const isDisabled =
+        queryConfig.searchAfter === undefined ||
+        total === 0 ||
+        queryConfig.firstPageFlag?.toString() === searchAfter?.tail?.toString();
```

Comment:
can be: `!queryConfig.searchAfter || !total || ... `
suggérer `=== undefined` ca fait peur: on se demande si `null` est ok ?
c'est plus explicite: ces values n existent pas ? c'est disable.
Aussi tu .toString() des tableaux pour les comparer c'est pas bon, il peut avoir la meme valeur mais pas au meme index.

Reply:
Je préfère priorisé `undefined` https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined
Car searchAfter n'est pas défini au premier appel à l'api, ce qui permet de le spécifier comme optionnel dans typescript.
Le toString est nécessaire car le tableau doit aussi être dans le même ordre pour ce cas, c'est relié à comment on fait le sort.